mail_notification_with_history/models/mail_thread.py

```diff
@@ -0,0 +1,14 @@
+# Copyright 2022 Camptocamp SA
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html)
+
+from odoo import api, models
+
+
+class MailThread(models.AbstractModel):
+    _inherit = "mail.thread"
+
+    _mail_notification_include_history = False
+
+    @api.model
+    def with_mail_notification_history(self):
+        return self._mail_notification_include_history
```

Comment:
```suggestion
```

I'd remove this and stick to using the property.
Properties can still be overriden, anyway.

btw, public model methods are exposed through xmlrpc.
Rule of thumb is keep everything private unless you really want this, IMO.

Reply:
I added that one when I was toying with the mocking in the tests. It is removed